src/main/java/io/xpring/xrpl/Utils.java

```diff
@@ -116,4 +120,97 @@ public static boolean isValidClassicAddress(String address) {
   public static String toTransactionHash(String transactionBlobHex) {
     return javaScriptUtils.toTransactionHash(transactionBlobHex);
   }
+
+  /**
+   * Convert from units in drops to xrp.
+   *
+   * @param drops An amount of xrp expressed in units of drops.
+   * @return A String representing the drops amount in units of xrp.
+   * @throws Exception if drops is in an invalid format.
+   */
+  public static String dropsToXrp(String drops) throws Exception {
```

Comment:
Generally it's not common to throw exceptions using the base `Exception` type. You want to pick something more specific. For all the exceptions that you are throwing in these methods,`IllegalArgumentException` would be more appropriate.